src/platform-includes/distributed-tracing/index/javascript.mdx

```diff
@@ -0,0 +1,3 @@
+All of Sentry's tracing-related integrations (`BrowserTracing`, `Http`, and `Express`), as well as the [Next.JS SDK](/platforms/javascript/guides/nextjs/), support distributed tracing out of the box.
```

Comment:
```suggestion
All of Sentry's tracing-related integrations (`BrowserTracing`, `Http`, and `Express`), as well as the [Next.JS](/platforms/javascript/guides/nextjs/) and [SvelteKit](/platforms/javascript/guides/sveltekit/) SDKs, support distributed tracing out of the box.
```

Reply:
Implemented the suggestion of @Lms24 everywhere (with a slight change in the links)